libs/node-sdk/src/utils/retry.ts

```diff
@@ -0,0 +1,24 @@
+const maxRetries = 5
+const initialDelayMs = 1000
+const backoffFactor = 2
+
+export async function retry<T>(
```

Comment:
I think the node-sdk can reference the JS SDK if we wanted to de-dupe (I think it does this for the FeatureBoard client interface?)